pkg/utils/restconfig/restconfig.go

```diff
@@ -0,0 +1,25 @@
+package restconfig
+
+import (
+	"context"
+
+	"k8s.io/client-go/rest"
+)
+
+type contextKey string
+
+const (
+	restKey contextKey = "restKey"
```

Comment:
consider a struct and this style: https://github.com/knative/pkg/blob/b558677ab03404ed118ba3e179a7438fe2d8509a/apis/contexts.go#L28

Reply:
Ok

Reply:
done